README.md

```diff
@@ -2398,8 +2405,139 @@ hazelcastClient.getMap('brothersMap').then(function (mp) {
+
+#### 7.8.2.2. Near Cache Example for Map
+
+**Declarative:**
+
+```
+{
+    "nearCaches": [
+        {
+            "name": "mostlyReadMap",
+            "inMemoryFormat": "object",
+            "invalidateOnChange": true,
+            "evictionPolicy": "lru",
+            "evictionMaxSize": 5000,
+        }
+    ]
+}
+```
+
+**Programmatic:**
+
+```javascript
+var nearCacheConfig = new Config.NearCacheConfig();
+nearCacheConfig.name = "mostlyReadMap";
+nearCacheConfig.inMemoryFormat= InMemoryFormat.OBJECT;
+nearCacheConfig.invalidateOnChange = true;
+nearCacheConfig.evictionPolicy = EvictionPolicy.LRU;
+nearCacheConfig.evictionMaxSize = 5000;
+
+cfg.nearCacheConfigs['mostlyReadMap'] = nearCacheConfig;
+```
+
+#### 7.8.2.3. Near Cache Eviction
+
+In the scope of Near Cache, eviction means evicting (clearing) the entries selected according to the given `evictionPolicy` when the specified `evictionMaxSize` has been reached.
+
+The `evictionMaxSize` defines the entry count when the Near Cache is full and determines whether the eviction should be triggered. 
+
+Once the eviction is triggered the configured `evictionPolicy` determines which, if any, entries must be evicted.
+
+#### 7.8.2.4. Near Cache Expiration
+
+Expiration means the eviction of expired records. A record is expired:
+
+- if it is not touched (accessed/read) for `maxIdleSeconds`
+
+- `timeToLiveSeconds` passed since it is put to Near Cache
+
+The actual expiration is performed in two cases:
+
+- When a record is accessed: it is checked if the record is expired or not. If it is expired, it is evicted and `undefined` is returned as the value to the caller.
+
+- In the background: there is an expiration task that periodically scans records and evicts the expired records.
+
+#### 7.8.2.5. Near Cache Invalidation
+
+Invalidation is the process of removing an entry from the Near Cache when its value is updated or it is removed from the original data structure (to prevent stale reads). See [Near Cache Invalidation section](https://docs.hazelcast.org/docs/latest/manual/html-single/#near-cache-invalidation) in the Hazelcast IMDG Reference Manual.
+
+#### 7.8.2.6. Near Cache Eventual Consistency
+
+Near Caches are invalidated by invalidation events. Invalidation events can be lost due to the fire-and-forget fashion of eventing system. If an event is lost, reads from Near Cache can indefinitely be stale.
+
+To solve this problem, Hazelcast provides eventually consistent behavior for Map Near Caches by detecting invalidation losses. After detection of an invalidation loss, stale data will be made unreachable and Near Cache’s `get` calls to that data will be directed to underlying Map to fetch the fresh data.
+
+You can configure eventual consistency with the `ClientConfig.properties` below:
+
+- `hazelcast.invalidation.max.tolerated.miss.count`: Default value is `10`. If missed invalidation count is bigger than this value, relevant cached data will be made unreachable.
```

Comment:
Node.js client keeps a count of missed invalidation events per map partition. If the total number of missed events reach this number, all the partitions that have at least one missing event are cleared.

Reply:
nevermind, my explanation was not correct at all. Yours is better